pkg/apis/zalando.org/v1/types.go

```diff
@@ -42,6 +42,10 @@ type ElasticsearchDataSetSpec struct {
 	// +optional
 	SkipDraining bool `json:"skipDraining"`
```

Comment:
can we make this property part of the draining config?

Reply:
That's a really good idea and I thought about it. The catch is how can we handle backward compatibility?

Reply:
@mikkeloscar you probably know: How do we evolve the CRD so that the propery moves into the new draining properties?

Reply:
Any update regarding this thread?

Reply:
I read about it, and we can deprecate the field, introduce a new one, and check the values. We can then decide based on different values being presented, and eventually remove the old field. I don't think it makes sense to do all this, but if that's your preference, I can send another commit making these changes.

Reply:
ok, let's park it for a follow-up.